packages/foam-vscode/src/test/suite-unit.ts

```diff
@@ -8,32 +8,25 @@
  *   they will make direct use of the vscode API to be invoked as commands, create editors,
  *   and so on..
  */
+
+process.env.FORCE_COLOR = '1';
+process.env.NODE_ENV = 'test';
+
```

Comment:
I moved `process.env.FORCE_COLOR = '1'` before the imports because in newer versions of foam it won't have any effect otherwise. This is due to an early import of chalk happening immediately after the import of jest (see my report https://github.com/facebook/jest/issues/12162). So this is future-proof(er). For the sake of consistency, I also moved the other environment setting here.

Reply:
Good catch! I would add this comment in the code, for future reference.

Reply:
Done.